project_event/models/res_partner.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2018 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _name = 'res.partner'
+    _inherit = ['res.partner']
+
+    client_type = fields.Many2one(
```

Comment:
this field should be related to the value of the client type of the category of the partner

Reply:
@rimbendhaou Changed